changelog/8-0-0-Release.md

```diff
@@ -0,0 +1,46 @@
+# 8.0.0 Release
+
+Clean up: 
+
+Before 8, all LST for a repository needed to be loaded into memory and available for LST to operate.  For large monorepos like Shelter’s ecommerce or Block’s franklin, this would not fit into memory. So we needed to redesign everything because the constraint now is that only part of LST is available in memory at a time and you don’t have random access - therefore LST needs to be scanned first to collect all info and then edit LST.
```

Comment:
Shelter’s ecommerce or Block’s franklin are internal customer information, I think we should hide this.